hedera-node/hedera-app-spi/src/testFixtures/java/com/hedera/node/app/spi/fixtures/Scenarios.java

```diff
@@ -341,15 +342,15 @@ default Map<AccountID, Account> defaultAccounts() {
                 ERIN.accountID(), ERIN.account());
     }
 
-    default Map<Bytes, AccountID> defaultAliases() {
+    default Map<ProtoBytes, AccountID> defaultAliases() {
```

Comment:
Why do we want to store `ProtoBytes` instead of `Bytes`?

Reply:
Oh, good question. Byte arrays are native to protobuf, do we need ProtoBytes?

Reply:
Here perhaps not (as it's not directly values in a KV State), but we cannot store Bytes directly in a KV State without a Codec, and there isn't one in PBJ for Bytes.  The ProtoBytes was created for the same reason as ProtoLong, ProtoString, etc...  It keeps everything in State as a Protobuf, rather than some exceptions.  Alias is, currently, a ByteString in state, so it's a google protobuf object that's sort of both a byte[] and a String.  It appears we always treat alias values as byte[] so this PR changes how it's stored in KV States from ProtoString to ProtoBytes.